memory.js

```diff
@@ -0,0 +1,70 @@
+var content = ['blk10','blk10','blkace','blkace','blkjack','blkjack','blkjoker','blkjoker','blkking','blkking','blkqueen','blkqueen','red10','red10','redace','redace','redjack','redjack','redjoker','redjoker','redking','redking','redqueen','redqueen'];
+var values = [];
+var card_ids = [];
+var cardsFlipped = 0;
+
+//Fisher Yates Shuffle Modern Algorithm
```

Comment:
Thanks for referencing the algorithm, but if you found the code somewhere, please include a link to it!

https://github.com/advanced-js/syllabus#instructor

Also, it's a good idea to avoid modifying JS objects that you don't "own".

https://www.nczonline.net/blog/2010/03/02/maintainable-javascript-dont-modify-objects-you-down-own/